package.json

```diff
@@ -20,8 +20,8 @@
     "test:unit": "cross-env NODE_ENV=test API_VERSION=${npm_package_version} jest --runInBand",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:unit:coverage": "cross-env npm run test:unit -- --coverage",
-    "test-i": "echo 'to be added: launch server' && npm run test:integration",
-    "test:integration": "cross-env NODE_ENV=test echo 'integration tests to be added'",
+    "test-i": "npm run test:integration",
```

Comment:
Can you make this follow the naming convention?

Reply:
Sorry @premghinde - could you elaborate please? I'm not sure how you want them changed. These scripts are from [the template repo](https://github.com/dvsa/cvs-svc-template)

Reply:
All other commands are separated with `:`'s creating a de facto naming standard. The hypenated one is the odd one out. I would suggest a fix in the CVS template also.

Reply:
Potential complication: If we did change it, we would need to update the pipeline too. We would then have to manage two versions until all repos have been converted.

Reply:
Please add a CI ticket for this

Reply:
@premghinde Task raised to create a CI ticket for the change requested: [CB2-2595](https://jira.dvsacloud.uk/browse/CB2-2595)